frontend/public/components/utils/dropdown.jsx

```diff
@@ -624,7 +624,12 @@ Dropdown.propTypes = {
 
 class ActionsMenuDropdown extends DropdownMixin {
   render() {
-    const { actions, title = undefined } = this.props;
+    const {
+      actions,
+      title = undefined,
+      actionsMenuClass = undefined,
+      toggleButtonClass = undefined,
+    } = this.props;
```

Comment:
I don't think any of the changes are needed in `dropdown.jsx` as we are not overloading the ActionsMenu (based on my comments).